core/server/master/src/test/java/alluxio/master/lineage/LineageMasterTest.java

```diff
@@ -48,7 +48,7 @@
 import java.util.concurrent.ThreadFactory;
 
 /**
- * Unit tests for {@link LineageMaster}.
+ * Unit tests for {@link DefaultLineageMaster}.
```

Comment:
can you revert all the change from `LineageMaster` to `DefaultLineageMaster` in Javadoc like this in this file?